### ISSUE TITLE
Video should have sound even iphone is silent from the side button

### ISSUE THREAD

**Issue by MrHazimAli** [Ziggeo/ReactNativeSDK#18]
Hye,
I would like to ask whether this functionality is supported by this package?

For example in youtube app, when I open the video and play it, then I silent the Iphone from the side button, the video sound still on and I wonder if I can do the same with this package using ziggeo as of right now, when I silent the Iphone using side button, my video inside the app also got silent.

Thanks!

**Comment by MrHazimAli** [comment]
@3akat , yes.. ios only.. I'm using version 1.2.0 but apparently, when the device is silent, the video from ziggeo also got silent. any additional setting that I need to do to ensure when the device is silent, the video sound should not be affected?

**Comment by 3akat** [comment]
@MrHazimAli don;t know yet, will discuss this with iOs dev

**Comment by MrHazimAli** [comment]
alright.. thanks again @3akat ! 👍

**Comment by MrHazimAli** [comment]
noted! thanks @3akat 👍

**Comment by 3akat** [comment]
Hi @MrHazimAli!
Please, take a look at the version 1.3.0.
